src/modules/75-cf/components/CreateEnvironmentDialog/__tests__/EnvironmentDialog.test.tsx

```diff
@@ -13,24 +13,51 @@ import * as useFeaturesMock from '@common/hooks/useFeatures'
 import * as usePlanEnforcementMock from '@cf/hooks/usePlanEnforcement'
 import { FeatureIdentifier } from 'framework/featureStore/FeatureIdentifier'
 import type { CheckFeatureReturn } from 'framework/featureStore/featureStoreUtil'
-
+import mockImport from 'framework/utils/mockImport'
 import EnvironmentDialog, { EnvironmentDialogProps } from '../EnvironmentDialog'
+import mockCreateEnvironmentResp from './mockCreateEnvironmentResp'
+
+const onCreate = jest.fn()
+const createEnvironment = jest.fn().mockResolvedValue(mockCreateEnvironmentResp)
 
 const renderComponent = (props: Partial<EnvironmentDialogProps> = {}): RenderResult => {
   return render(
     <TestWrapper
       path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/feature-flags"
       pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
     >
-      <EnvironmentDialog onCreate={jest.fn()} environments={[]} {...props} />
+      <EnvironmentDialog onCreate={onCreate} environments={[]} {...props} />
```

Comment:
Looks like there are no assertions against this, so might as well inline the default
```suggestion
      <EnvironmentDialog onCreate={jest.fn()} environments={[]} {...props} />
```